packages/language/test/utils.ts

```diff
@@ -18,6 +18,22 @@ interface AssertNoDiagnosticsOptions {
   ignoreSeverity?: Severity[];
 }
 
+const SyntaxKindReverseLookup: Map<SyntaxKind, string> = new Map(
+  Object.values(SyntaxKind)
+    .filter((key) => typeof key === "number")
+    .map((key) => [key, SyntaxKind[key]]),
+);
+
+/**
+ * This function assigns a `_kind: string` to each node that
+ * alleviates debugging by allowing the user to see the kind
+ * of the node in the debugger.
+ */
+export function assignDebugKind(node: SyntaxNode) {
+  (node as any)._kind = SyntaxKindReverseLookup.get(node.kind);
```

Comment:
Why not call it `_debugKind`? In case we see if somewhere else that would be a dead giveaway that it's somewhere we don't expect it to be.